meta/documents/changelog_de.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes für Kauf auf Rechnung
 
+## 2.0.5 (2020-02-28)
+### Geändert
+- Code Optimierung um viele Datenbankabfragen zu vermeiden.
```

Comment:
Wäre ein Kunden-Changelog nicht verständlicher, wenn man etwas in Richtung:
```
Performance-Optimierung für den Ladevorgang der Plugin-Einstellungen und verfügbaren Lieferländer.
```
(Man kann ja nicht davon ausgehen, dass ein Kunde auch zwangsläufig etwas mit der Aussage: Vermeidung überflüssiger Datenbankabfragen etwas anfangen kann)
Ggf. könnten wir am Montag auch nochmal mit Jaqueline darüber diskutieren, oder andere Meinungen einholen.